client/components/spinner-button/index.jsx

```diff
@@ -51,7 +51,8 @@ export default React.createClass( {
 				{ loading &&
```

Comment:
> Was there a design reference for this?

The spinner button was built from the early designs for the exporter where it's also used (p195om-2Bg-p2):
![screen shot 2016-09-01 at 1 37 06 pm](https://cloud.githubusercontent.com/assets/416133/18154587/38b2523a-7049-11e6-9cc3-bd339b511f9c.png)

Though these designs are over a year old now - @rickybanister any thoughts on changing this spinner?

I'll leave this as is for now so we can launch GT, but if there's a design improvement we can make I'll put together a separate PR to modify it.